src/components/event/ConferenceSubmissionForm.vue

```diff
@@ -278,3 +280,11 @@ export default defineComponent({
     }
 });
 </script>
+
+<style scoped>
+
+.serial-event {
+    margin-left: 10px;
+}
+
+</style>
```

Comment:
Is this a good practice to use scoped style? It might be complicate for redesigning the UI. Can we move this into css file?

Reply:
I must consult with Milos about this as he is more experienced with CSS. This is a Vue feature and I haven't seen anywhere that people suggest not using it, I can research this a little bit and maybe we can discuss this in a meeting?